### PR TITLE
Add syntax highlighting and misc changes

### PR DESCRIPTION
**Pull request by OmgImAlexis**
I found the `README.md` much easier to read with these changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,61 +7,61 @@
...
 npm install execution-time --save
 ```
 
 ## Usage
-```
+```js
 const perf = require('execution-time')();
 
-//at beginning of your code
+// At beginning of your code
```
Comment (sampathBlam) [inline README.md]:
```suggestion
// At the beginning of your code
```

File: README.md [hunk 2]
```diff
@@ -7,61 +7,61 @@
...
 perf.start();
 
-//at end of your code
+// At end of your code
```
Comment (sampathBlam) [inline README.md]:
```suggestion
// At the end of your code
```

File: README.md [hunk 3]
```diff
@@ -7,61 +7,61 @@
...
 console.log(results.time);  // in milliseconds
 ```
 
 ### Named performance measurer
-```
+```js
 const perf = require('execution-time')();
 
-//at beginning of your code, pass any name
+// At beginning of your code, pass any name
```
Comment (sampathBlam) [inline README.md]:
```suggestion
// At the beginning of your code, pass any name
```

File: README.md [hunk 4]
```diff
@@ -7,61 +7,61 @@
...
 perf.start('apiCall');
 
-//at end of your code, pass the same name (anywhere in your flow)
+// At end of your code, pass the same name (anywhere in your flow)
```
Comment (sampathBlam) [inline README.md]:
```suggestion
// At the end of your code, pass the same name (anywhere in your flow)
```

File: README.md [hunk 5]
```diff
@@ -7,61 +7,61 @@
...
 const results = perf.stop('apiCall');
 console.log(results.time);  // in milliseconds
 console.log(results.preciseWords);  // in words
 
 
 ```
 
-### if you dont care about results but just want to print results
+### If you dont care about results but just want to print results
```
Comment (sampathBlam) [inline README.md]:
```suggestion
### If you don't care about results but just want to print results
```

File: README.md [hunk 6]
```diff
@@ -7,61 +7,61 @@
...
 
-sample usage with debug
-```
+Sample usage with debug
+```js
 const debug = require('debug')('your-module');
 const perf = require('execution-time')(debug);
 
-//at beginning of your code
+// At beginning of your code
```
Comment (sampathBlam) [inline README.md]:
```suggestion
// At the beginning of your code
```

File: README.md [hunk 7]
```diff
@@ -7,61 +7,61 @@
...
 perf.start();
 
-//at end of your code
+// At end of your code
```
Comment (sampathBlam) [inline README.md]:
```suggestion
// At the end of your code
```

File: README.md [hunk 8]
```diff
@@ -7,61 +7,61 @@
...
 ```
 
 also works with console.log/info/warn
 
-```
+```js
 const perf = require('execution-time')(console.log);
 
-//at beginning of your code
+// At beginning of your code
```
Comment (sampathBlam) [inline README.md]:
```suggestion
// At the beginning of your code
```

File: README.md [hunk 9]
```diff
@@ -7,61 +7,61 @@
...
 perf.start();
 
-//at end of your code
+// At end of your code
```
Comment (sampathBlam) [inline README.md]:
```suggestion
// At the end of your code
```

**Comment by OmgImAlexis** [comment]
If you want those changes then go ahead and open a PR yourself.

**Comment by sampathBlam** [comment]
@OmgImAlexis : You can always ignore requested changes in Github.

**Comment by OmgImAlexis** [comment]
Or you could not fly by random PRs and dump changes like this.

**Comment by svenkatreddy** [comment]
Thanks for opening this PR.
Looks good. Merged.

**Pull request merged**